tests/benchdnn/inputs/matmul/test_matmul_ci

```diff
@@ -211,3 +211,11 @@
 
 # fp4
 --batch=test_matmul_fp4
+
```

Comment:
Can you add a test case covering the change enable f32bf16fp32 datatypes i.e with --dt=f32:bf16:f32 ? Could you also paste the verbose output before and after the change?

Reply:
Approved based on https://review.mlplatform.org/c/ml/ComputeLibrary/+/13341 getting merged.